lib/modules/datasource/docker/index.ts

```diff
@@ -1,4 +1,3 @@
-import URL from 'node:url';
 import { ECR } from '@aws-sdk/client-ecr';
```

Comment:
The word `deprecated` in the PR title has a typo.

```diff
- refactor(datasource/docker): drop node:url because it is depcrecated
+ refactor(datasource/docker): drop deprecated node:url
```

You could also shorten the title a bit and keep the same meaning.